library/std/src/panicking.rs

```diff
@@ -246,7 +246,10 @@ fn default_hook(info: &PanicInfo<'_>) {
 pub fn panic_hook_with_disk_dump(info: &PanicInfo<'_>, path: Option<&crate::path::Path>) {
     // If this is a double panic, make sure that we print a backtrace
     // for this panic. Otherwise only print it if logging is enabled.
-    let backtrace = if panic_count::get_count() >= 2 {
+    // We do not keep printing for further panics, so that e.g. a "panic inside a drop on an unwind
+    // path", which leads to a triple-panic (the third panic being "cannot unwind here"), doesn't
+    // print *two* backtraces.
+    let backtrace = if panic_count::get_count() == 2 {
```

Comment:
For panic count > 2 should this return `None`?

Reply:
Should what return None? The function we are in does not return anything.

Reply:
I meant the `backtrace` variable should be assigned `None` if the panic count > 2. This seems to indicate that backtrace printing is not available.

Reply:
But why would we want that? I think we still want RUST_BACKTRACE to be respected.

Reply:
For a triple panic I can imagine there is a fair chance that the panic actually happens while printing the backtrace and thus we shouldn't try to print a backtrace for the third panic.

Reply:
We have a triple-panic already when just causing a panic during cleanup. So I don't think that this is a good indication for panicking during backtrace printing.